### PR TITLE
Addition of Calypso NFT Hub | SKALE Chain

### PR DESCRIPTION
**Pull request by TheGreatAxios**
# Calypso NFT Hub
The Calypso NFT Hub is a chain apart of the multi-chain SKALE Network. This chain is part of the community hub model. 
It is designed to be shared by many dApps in order to best utilize resources in addition to optimizing user experience.

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Please consider contributing to our grant: https://gitcoin.co/grants/2974/ethereum-lists
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by milad237** [comment]
> # Calypso NFT Hub
> The Calypso NFT Hub is a chain apart of the multi-chain SKALE Network. This chain is part of the community hub model. 
> It is designed to be shared by many dApps in order to best utilize resources in addition to optimizing user experience. 
>

**Comment by milad237** [comment]
- [ ]

File: _data/chains/eip155-1564830818.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  "faucets": [
+    "https://sfuel.dirtroad.dev"
+  ],
+  "nativeCurrency": {
+    "name": "sFUEL",
+    "symbol": "sFUEL",
+    "decimals": 18
+  },
+  "infoURL": "https://calypsohub.network/",
```
Comment (milad237) [inline _data/chains/eip155-1564830818.json:16]: - [ ]

**Review by milad237 — APPROVED** [review]

**Review by kamikase2005 — APPROVED** [review]

**Comment by TheGreatAxios** [comment]
@ligi Ran the prettier function as requested.
Hopefully that fixes this issue!

**Comment by TheGreatAxios** [comment]
@milad237 @kamikase2005 I pushed the changes as requested. 
Can you please re-review. 

Thank you!

**Comment by TheGreatAxios** [comment]
@ligi Not sure why another repo was tagged above, but can you please confirm whether this can be merged or not? 

Thanks!

**Review by ligi — APPROVED** [review]

**Pull request merged**